src/modules/map/components/openlayers/utils/usePrintAreaRenderer.composable.js

```diff
@@ -118,6 +118,21 @@ export default function usePrintAreaRenderer(map) {
         const width = size[0]
 
         const printRectangle = calculatePageBoundsPixels(selectedScale.value, printLayoutSize.value)
+        const topLeftCoordinate = map.getCoordinateFromPixel([printRectangle[0], printRectangle[1]])
+        const rightBottomCoordinate = map.getCoordinateFromPixel([
+            printRectangle[2],
+            printRectangle[3],
+        ])
+
+        store.commit('setPrintExtent', {
+            printExtent: [
+                topLeftCoordinate[0],
+                rightBottomCoordinate[1],
+                rightBottomCoordinate[0],
+                topLeftCoordinate[1],
+            ],
+            dispatcher,
+        })
```

Comment:
I don't know if this would help, but there are a couple extent utils in the file `src/utils/coordinateUtils.js` that might be of use here.
There's a function to flatten an extent that is expressed as `[ [x0, y0], [x1, y1] ]` and the opposite to make it a double coordinate array

If you want to use them, you could move these two function in the newly created `src/utils/extentUtils.js` file instead

Reply:
It's different I guess, since this code compute the coordinate-based extent from the pixel-based extent. The order is in different direction (e.g. pixel is from top-left while the coordinate is from left-bottom).

If it's ok, I will keep it in this location because it's very specific. I have added a comment to explain the order.